pkg/operator/controller/externaldns/deployment_test.go

```diff
@@ -2771,6 +2787,7 @@ func TestEnsureExternalDNSDeployment(t *testing.T) {
 							BlockOwnerDeletion: &test.TrueVar,
 						},
 					},
+					Annotations: map[string]string{"external-dns-credentials-test": ""},
```

Comment:
Can you please add a test case with an actual secret in `existingObjects` so that the hash calculation is tested?

Reply:
Resolved